_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+
+In this post we are going to follow the steps necessary to configure OpenFaaS to execute function on AWS Lambda using 
+the new faas-lambda provider. See the previous blog post [announcing support for AWS Lambda] for an overview of the 
+technology.
+
+
+### Pre-requisites
+To follow the steps in this article it is only necessary to have a Kubernetes cluster with Helm installed.
+
+- [Kubernetes cluster](https://kubernetes.io/docs/setup/)
+- [Helm installed](https://github.com/openfaas/faas-netes/blob/master/HELM.md)
+
+### 1. Get an early access token
+`faas-lambda` is currently in early access, please you the link below to get your access token
+
+- [Create an access token](https://ewilde.o6s.io/faas-lambda-jwt-page)
+
+### 2. Deploy OpenFaaS faas-lambda using helm
+
+Installing OpenFaaS for Lambda requires the following high-level steps:
+
+- Create a Kubernetes namespaces `openfaas` for the OpenFaaS system components.
+- Create AWS IAM roles. One for the provider and one for deployed functions to inherit. 
+- Allow the provider access to AWS using access key and secret, which are stored as Kubernetes secretes
+- Deploy the OpenFaaS Helm chart using specific Lambda configuration options
+
+#### 2.1. Create recommended namespaces
+`$kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml`
+
+#### 2.2. Create an OpenFaaS Provider IAM role
+In order for the `faas-lambda` provider to communicate with the AWS Lambda service is needs the following minimum 
+permissions:
+
+**Minimum persmissions**
+
+```javascript
+
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "LambdaPermissions",
+            "Effect": "Allow",
+            "Action": [
+                "lambda:CreateFunction",
+                "lambda:DeleteFunction",
+                "lambda:GetAccountSettings",
+                "lambda:GetAlias",
+                "lambda:GetEventSourceMapping",
+                "lambda:GetFunction",
+                "lambda:GetFunctionConfiguration",
+                "lambda:GetLayerVersion",
+                "lambda:GetLayerVersionPolicy",
+                "lambda:GetPolicy",
+                "lambda:InvokeFunction",
+                "lambda:InvokeAsync",
+                "lambda:ListAliases",
+                "lambda:ListEventSourceMappings",
+                "lambda:ListFunctions",
+                "lambda:ListLayers",
+                "lambda:ListLayerVersions",
+                "lambda:ListTags",
+                "lambda:ListVersionsByFunction",
+                "lambda:TagResource",
+                "lambda:UpdateFunctionCode",
+                "lambda:UpdateFunctionConfiguration"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "IAMPermissions",
+            "Effect": "Allow",
+            "Action": [
+                "iam:PassRole"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+
+##### 2.2.1. Recommended setup
```

Comment:
Is there value in a screenshot here?

Reply:
Possibly what did you have in mind? I can't think of anything that summarizes that IAM configuration that you could put in a screenshot